include/alpaka/rand/RandGenericSycl.hpp

```diff
@@ -0,0 +1,238 @@
+/* Copyright 2022
+ *
+ * This file is part of alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
We no longer do this. Please use SPDX identifiers:

```suggestion
 * SPDX-License-Identifier: MPL-2.0
```